extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -108,24 +116,31 @@ function process_anchor_params() {
 								'title' => $track['title'],
 							),
 						);
+
+						// Add insert basic template action.
+						$data['actions'][] = array(
+							'insert-episode-template',
```

Comment:
I guess we can remove the `set-episode-title` action now, and set the post title as part of the `insert-episode-template` action.

Reply:
Well... I think it's ok keeping these tasks in different actions. The actions system looks good to me, and clearly, they are different actions. Why do you think we should move it?

Reply:
> Why do you think we should move it?

Basically because they both are performed under the same conditions, so the resulting code of merging the two actions could be a bit simpler. But yeah, not a big deal, we can keep them separated.